test/sprinkler/watering_policy_test.exs

```diff
@@ -0,0 +1,72 @@
+defmodule Sprinkler.Devices.WateringPolicyTest do
+  use Sprinkler.DataCase
+
+  # alias Sprinkler.Devices.GardenReading
```

Comment:
Lol, you can uncomment this. I think the code would look much better with this alias